test/e2e/run.sh

```diff
@@ -765,6 +770,30 @@ verify() { # script API
     #          nodes and that pod0c0 is allowed to run on 4 CPUs:
     #   verify 'set.intersection(nodes["pod0c0"], nodes["pod1c0"]) == set()' \
     #          'len(cpus["pod0c0"]) == 4'
+    local retries=0
```

Comment:
We need to have default re-try count in order to fight flakyness
```suggestion
    local retries=3
```